jenkins/image_building/upload-ci-image.sh

```diff
@@ -68,8 +80,18 @@ upload_ci_image_xerces() {
   export OS_USER_DOMAIN_NAME="xerces"
   export OS_IDENTITY_API_VERSION=3
 
+ # Check if the common image already exists
+  if openstack image show "${COMMON_IMAGE_NAME}" &>/dev/null; then
```

Comment:
Have you checked, does this return non zero if it can't find any image?

Reply:
Yes